style.css

```diff
@@ -0,0 +1,71 @@
+html,
+body {
+  height: 100%;
+  background: url("images/header_bg.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center center;
+  background-size: 110%;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  justify-content: flex-end;
+  padding-left: 3%;
+  padding-right: 5%;
+  margin-top: 215px;
+}
+
+.header-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  color: #172b4d;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin-bottom: 7px;
+}
```

Comment:
- [ ] Kindly use the right `font-family`  as seen in your chosen `Figma` design 🙏